ecmascript/parser/src/parser/stmt.rs

```diff
@@ -1837,6 +1837,24 @@ export default function waitUntil(callback, options = {}) {
         );
     }
 
+    #[test]
+    #[should_panic(expected = "Expected ,, got b")]
```

Comment:
Can you add tests to https://github.com/swc-project/swc/tree/83d88ce388ee8553af6507032630ddbc7a100d14/ecmascript/parser/tests/typescript-errors instead?

Reply:
Should these tests be removed?

Reply:
I think it's better, as it's a duplicate.